bugsnag/django/__init__.py

```diff
@@ -45,14 +46,23 @@ def add_django_request_to_notification(notification):
 
     if getattr(request, "session", None):
         notification.add_tab("session", dict(request.session))
-    notification.add_tab("request", {
+    request_tab = {
         'method': request.method,
         'path': request.path,
         'encoding': request.encoding,
         'GET': dict(request.GET),
         'POST': dict(request.POST),
         'url': request.build_absolute_uri(),
-    })
+    }
+    try:
+        if request.META.get('CONTENT_TYPE') == 'application/json':
```

Comment:
I think there's a few cases where this check won't work out properly given the spec (https://tools.ietf.org/html/rfc7231#section-3.1.1.5).

1. Media Type may contain parameters, for example: `application/json; charset=utf-8`
2. [RFC 7231 3.1.1.1](https://tools.ietf.org/html/rfc7231#section-3.1.1.1) specifies case-insensitive comparisons for the type/subtype components. I.e, `Application/JSON` should be equivilent to `application/json` for purposes of comparison here.
    > The type, subtype, and parameter name tokens are case-insensitive
3. Should this match against subtypes when there is a suffix? i.e, content-type such as `application/schema+json`

Reply:
Maybe `if request.META.get('CONTENT_TYPE').lower().startswith('application/json')` ?